docs/api.rst

```diff
@@ -119,6 +118,14 @@ Importation tools
 
     envs.gym_make
 
+StableBaselines3 Agent
+-----------------------
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
+   agents.StableBaselinesAgent
+
```

Comment:
I would have moved this to line 51 with something like 

```
   agents.torch.AVECPPOAgent
   agents.torch.REINFORCEAgent

 Importation tools
-----------------------
.. autosummary::
   :toctree: generated/
   :template: class.rst

   agents.StableBaselinesAgent
``` 
Jax Agents (experimental)
--------------------

Reply:
Maybe we should put it before the Torch agents for now (since they'll be released as experimental). What do you think?

Reply:
Yes you are right.

Reply:
Also, about the pre-commit you're right - it was caused by me updating one of the `__init__.py`'s, but I used `--no-verify` to commit anyways. We can create another PR to add the ignores later.